_posts/2021-02-01-coalesce-dbt-summary.markdown

```diff
@@ -0,0 +1,288 @@
+
+In brief, this presentation focuses on finding the hallowed nexus - of Effective Data Communication - between
+
+1. Engineering Practice
+1. Stakeholder Involvement
+1. Company Discoverability (of data) 
+
+Despite being a shorter presentation, this one was not short of insights - I took away some great reminders
+
+* Taxonomy of terms - speaking the same language - acts as an essential foundation for common understanding across disparate teams
+* Ensuring your stakeholder group can adequately critique your assumptions
+* Data teams should seek to remove themselves as an intermediary in discussions around data
+
+I liked the reference to [DBT Exposures](https://docs.getdbt.com/docs/building-a-dbt-project/exposures/) and how they can help cross-reference the indirect way in which modelled data is used. 
+
+### How to build a reputation on more than just Dashboards
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/Kub2bXrKmOE" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+Before the first 30 seconds are up, Andrea Kopitz sets out the aspirational goal
+
+> *I believe the best data nerds and data teams are the ones that always try to align themselves with the goals and interests of the business*
+
+In this super-concise but jam-packed presentation, Andrea illustrates the value data teams can offer organisations beyond simply cranking out dashboards. Along recurring themes of growing to be a partner to the business as opposed to servicing dashboard requests, she gives many great examples to assist that journey of growth
+
+* Facilitating communication through a single, common and accessible channel to promote and encourage interaction and inquisition
+* Consider every business request an opportunity to learn more about business domains
+* Anchor the knowledge gleaned into documentation to accelerate shared learning and growth
+* Set context around dashboards to manage expectations and state original purpose and assumptions / caveats
+
+In particular, I liked the suggestion to encourage and collaborate with early adopters, a formulate I've seen work well and also echoed in JetBlue's transition to a data-driven organisation.
+
+
+## Summary
+
+I came away invigorated after watching Coalesce 2020, not only about the topics and community around DBT, but of the future and rapid growth we will see in data; it's often easy to imagine this dystopically in light of data mining and privacy violations, however it inspired in me a more local, contextualized growth opportunity for companies to understand better what makes them tick, and what really determines their success, based on data instead of *gut feel*. I truly believe DBT and the wider movements around ELT are making these capabilities more accessible than ever for businesses of all shapes and sizes.
```

Comment:
I'd separate the paragraph after the first ;  
don't caveat your invigoration.
i.e. 
"I came away invigorated after watching Coalesce 2020, not only about the topics and community around DBT, but of the future and rapid growth we will see in data.

It's often easy to imagine this dystopically in light of data mining and privacy violations, however it inspired in me a more local, contextualized growth opportunity for companies to understand better what makes them tick, and what really determines their success, based on data instead of *gut feel*."